common.py

```diff
@@ -0,0 +1,72 @@
+import os
+from dotenv import load_dotenv
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+from webdriver_manager.chrome import ChromeDriverManager
+from selenium.webdriver.support import expected_conditions
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.common.by import By
+
+
+# Load env variables from ".env" file in the same folder
+load_dotenv(override=True)
+
+# Your EquatePlus username
+USERNAME = os.getenv("USERNAME")
+
+# Your EquatePlus password
+PASSWORD = os.getenv("PASSWORD")
+
+# DTC number for brokerage
+DTC = os.getenv("DTC")
+
+# Account number
+ACCOUNT_NUMBER = os.getenv("ACCOUNT_NUMBER")
+
+
+def initialize_driver():
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless=new")
+    # Automatically get and cache the webdriver for Chrome
+    driver = webdriver.Chrome(
+        service=Service(ChromeDriverManager().install()), options=options
+    )
+    driver.get("http://www.na.equateplus.com/")
+    return driver
+
+
+def initialize_wait(driver):
+    return WebDriverWait(driver, 10)
+
+
+def sign_in(wait, driver):
```

Comment:
I always get the splash page to get through each time. Do you? If not, I'll add in my changes as a conditional to fix what I see on my end.